cmd/cdi/cmd/validate.go

```diff
@@ -26,22 +26,23 @@ import (
 	"tags.cncf.io/container-device-interface/pkg/cdi"
 )
 
-// validateCmd is our CDI command for validating CDI Spec files in the registry.
+// validateCmd is our CDI command for validating CDI Spec files in the cache.
 var validateCmd = &cobra.Command{
 	Use:   "validate",
-	Short: "List CDI registry errors",
+	Short: "List CDI cache errors",
 	Long: `
 The 'validate' command lists errors encountered during the population
-of the CDI registry. It exits with an exit status of 1 if any errors
-were reported by the registry.`,
+of the CDI cache. It exits with an exit status of 1 if any errors
+were reported by the cache.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		cdiErrors := cdi.GetRegistry().GetErrors()
+		cache := cdi.GetDefaultCache()
+		cdiErrors := cache.GetErrors()
 		if len(cdiErrors) == 0 {
-			fmt.Printf("No CDI Registry errors.\n")
+			fmt.Printf("No CDI Cache errors.\n")
```

Comment:
Apparently the original code was inconsistent in this regard, but maybe we could fix it and should always use either lowercase `cache` or capitalized `Cache` in help messages and error/info strings in the `cdi` command. This PR already mostly uses lowercase, but this is an exception here.

Reply:
Thank you for the review! Should be fixed now.